keel-core/src/main/kotlin/com/netflix/spinnaker/keel/constraints/DependsOnConstraintEvaluator.kt

```diff
@@ -38,6 +40,27 @@ class DependsOnConstraintEvaluator(
       artifact,
       version,
       requiredEnvironment.name
+    ) && allVerificationsSucceededIn(
```

Comment:
can we add a log statement here, in case the verification failed?

Reply:
or even in both cases (succeeded/failed)

Reply:
Added!